app/main.py

```diff
@@ -1,2 +1,86 @@
 class Dictionary:
+
+    def __init__(self):
+        self.capacity = 8
+        self.threshold = int(self.capacity * (2 / 3))
+        self.length = 0
+        self.base_data = [None for i in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+
+        if self.length >= self.threshold:
+            self.resize()
+
+        index_ = int(hash(key) % self.capacity)
+
+        while self.base_data[index_] is not None:
+
+            if self.base_data[index_][0] == key and\
+                    self.base_data[index_][1] == hash(key):
+                self.base_data[index_] = [key, hash(key), value]
+                self.length -= 1
+                break
+            index_ = (index_ + 1) % self.capacity
+
+        self.base_data[index_] = [key, hash(key), value]
+        self.length += 1
+
+    def __getitem__(self, key):
+        index_ = hash(key) % self.capacity
+        while self.base_data[index_] is not None\
+                and (index_ < self.capacity):
```

Comment:
```suggestion
                and (index_ < self.capacity):
```
index_ won't reach `self.capacity` value because you take the remainder of the division in line 36.